vendor/src/github.com/docker/libnetwork/.gitignore

```diff
@@ -8,6 +8,8 @@ bin/
 integration-tmp/
 _obj
 _test
+.vagrant
```

Comment:
Not sure why vendor.sh picked it.
That would be a required change in that script.